searx/static/themes/__common__/less/stats.less

```diff
@@ -1,60 +1,54 @@
-.issue-hide{
-    display: none;
-}
-
-#main_stats {
-
-    table {
-	margin: 0 auto 0 0;
-    }
+.engine-stats {
 
-    table .engine-name {
-	width: 20rem;
+    .engine-name {
+        width: 20rem;
     }
 
-    table .engine-scores {
-	width: 7rem;
+    .engine-score {
+        width: 7rem;
+        text-align: right;
     }
 
-    table .result-count {
+    .result-count {
     }
 
-    table .response-time {
+    .response-time {
     }
 
-    table .engine-reliability {
-	text-align: right;
+    .engine-reliability {
+        text-align: right;
     }
 
     table.engine-error {
-	max-width: 1280px;
-	margin: 1rem;
-	border: 1px solid gray;
+        max-width: 1280px;
+        margin: 1rem;
+        border: 1px solid gray;
     }
 
     table.engine-error th.engine-error-type,
     table.engine-error td.engine-error-type,
     failed-test {
-	width: 10rem;
+        width: 10rem;
     }
 
     table.engine-error span.log_parameters
     {
-	border-right: 1px solid gray;
-	padding: 0 1rem 0 0;
-	margin: 0 0 0 0.5rem;
+        border-right: 1px solid gray;
+        padding: 0 1rem 0 0;
+        margin: 0 0 0 0.5rem;
     }
 }
 
-@media screen and (max-width: 75em) {
-}
-
 .bar-chart-value {
     width: 3em;
+    display: inline-block;
+    text-align: right;
+    padding-right: 0.5rem;
 }
 
 .bar-chart-graph {
-    width: 300px;
+    width: calc(100% - 5rem);
+    display: inline-block;
```

Comment:
From line 44 to line 51.

There is a slight visual change with this PR:
![image](https://user-images.githubusercontent.com/1594191/118454524-c1881e00-b6f8-11eb-86b2-3fd4f812aa7b.png)

The layout in the master branch is:
![image](https://user-images.githubusercontent.com/1594191/118454584-d2d12a80-b6f8-11eb-970a-66997e28096f.png)

Reply:
I haven't found the solution,  thanks a lot for this fix.